tests/unit/amo/sagas/test_home.js

```diff
@@ -421,6 +444,41 @@ describe(__filename, () => {
       expect(expectedAction).toEqual(errorAction);
     });
 
+    it('aborts fetching for a failed search fetch', async () => {
+      const state = sagaTester.getState();
+
+      const slug = 'collection-slug';
+      const userId = 123;
+
+      mockHeroApi
+        .expects('getHeroShelves')
+        .returns(Promise.resolve(createHeroShelves()));
+
+      const firstCollection = createFakeCollectionAddonsListResponse();
+      mockCollectionsApi
+        .expects('getCollectionAddons')
+        .withArgs({
+          api: state.api,
+          slug,
+          userId,
+        })
+        .returns(Promise.resolve(firstCollection));
+
+      const error = new Error('some API error maybe');
+
+      mockSearchApi
+        .expects('search')
+        .exactly(5)
```

Comment:
You can remove this since it won't have an affect (`mockSearchApi.verify()` is not called). It's not important for this test, anyway, which is about handling errors.